text/0000-startup-initialized-statics.md

```diff
@@ -0,0 +1,97 @@
+- Start Date: 2016-07-12
+- RFC PR #: 
+- Rust Issue #: 
+
+# Summary
+
+Introduce the ability to initialize (i.e. mutate) static items (even non-mut ones) at the beginning of main in a compiler-guaranteed safe manner.
+
+# Motivation
+
+Even though mutable statics can be initialized safely at runtime in a couple of ways, they have drawbacks:
+
+- By using the lazy initialization idiom. Drawback: incurs a small performance overhead by having to check that the item is initialized every time it is used.
+- By using the RAII idiom in the form of initialization guards, where the existence of an initialization guard variable is a guarantee that the initialization has happened. Drawback: very poor ergonomics.
+
+Instead, I propose a true zero-cost initialization scheme that is also ergonomic.
```

Comment:
It's not actually zero-cost since the initializer function will be invoked on startup. I guess you meant that all accesses are zero-cost.